OurTimeForTech/shiftwork2
# Set conditional workers views and add bio

## What type of PR is this? (check all applicable)

- [] Refactor
- [] Bug Fix
- [] Optimization
- [] Documentation Update
- [] Other (describe: )


## Description of what PR does
Adds the new bio field to worker show, view, edit. When users try to view a worker's page, they will only see worker first name, last name, and bio.

Gave the bio view a really simple UI, open to any suggestions for formatting. 

## Related PRs and/or Issues (if any)
- Completes step 3 and 4 of #52 
- Blocked by #64


## QA Instructions, Screenshots, Recordings
Log in as a worker to make sure you can see your shifts when viewing your own worker page, but only see name and bio when viewing another worker.
<img width="734" alt="Screen Shot 2021-04-29 at 2 57 49 PM" src="https://user-images.githubusercontent.com/29789422/116623570-4b637900-a8fb-11eb-8fef-eb4106e04f7e.png">


## Added tests?

- [x ] no, because they aren't needed for views


## Added to documentation?

- [x ] No documentation needed
